test/subscriber.ts

```diff
@@ -371,13 +377,12 @@ describe('Subscriber', () => {
 
   describe('getClient', () => {
     it('should get a subscriber client', async () => {
-      const pubsub = subscription.pubsub;
+      const pubsub = subscription.pubsub as {} as FakePubSub;
```

Comment:
Why do we need to cast this?

Reply:
to get access to client, as its a type of FakePubsub and client property is not available in actual pubsub.